components/_global-navigation.scss

```diff
@@ -171,7 +170,7 @@ $notification-counter-size: 18px;
 		// has `width` set in `%` and is inside `flexbox`.
 		min-width: 0;
 		padding: 3px 4px;
-		width: 100%;
+		width: 0;
```

Comment:
This element has padding that we don't hide on tablet breakpoint - it ends up with more space between search icon and user menu.